src/lib/bootloader/grub2base.rb

```diff
@@ -1,51 +1,59 @@
 # encoding: utf-8
 require "yast"
+require "bootloader/sections"
 require "bootloader/grub2pwd"
 require "bootloader/udev_mapping"
-
-module Yast
+require "bootloader/serial_console"
+require "config_files/grub2/default"
+require "config_files/grub2/grub_cfg"
+require "config_files/matcher"
+require "config_files/placer"
+
+Yast.import "Arch"
+Yast.import "BootArch"
+Yast.import "BootCommon"
+Yast.import "BootStorage"
+Yast.import "HTML"
+Yast.import "Initrd"
+Yast.import "Kernel"
+Yast.import "Mode"
+Yast.import "Pkg"
+Yast.import "Product"
+Yast.import "ProductFeatures"
+Yast.import "Stage"
+Yast.import "Storage"
+Yast.import "StorageDevices"
+
+module Bootloader
   # Common base for GRUB2 specialized classes
-  class GRUB2Base < Module
+  class GRUB2Base < BootloaderBase
+    include Yast::Logger
+    include Yast::I18n
+
     # @!attribute password
     #    @return [::Bootloader::GRUB2Pwd] stored password configuration object
     attr_reader :password
+    # @!attribute grub_default
+    #    @return [CFA::Grub2::Default] grub2 configuration object
+    attr_reader :grub_default
 
-    def main
-      Yast.import "UI"
-
+    def initialize
       textdomain "bootloader"
-
-      Yast.import "Arch"
-      Yast.import "BootArch"
-      Yast.import "BootCommon"
-      Yast.import "BootStorage"
-      Yast.import "HTML"
-      Yast.import "Initrd"
-      Yast.import "Kernel"
-      Yast.import "Mode"
-      Yast.import "Pkg"
-      Yast.import "Product"
-      Yast.import "ProductFeatures"
-      Yast.import "Stage"
-      Yast.import "Storage"
-      Yast.import "StorageDevices"
-
-      # includes
-      # for simplified widgets than other
-      Yast.include self, "bootloader/grub2/dialogs.rb"
-
       @password = ::Bootloader::GRUB2Pwd.new
+      @grub_default = ::ConfigFiles::Grub2::Default.new
```

Comment:
It seems that ConfigFiles was renamed to CFA (in <https://github.com/config-files-api>), right?